index.html

```diff
@@ -65,43 +85,72 @@
                   <a href="#{{category.name | nospace}}" ng-click="categoryFilter($event)" id="{{category.name | nospace}}">
                     <div class="flex-container" id="{{category.name | nospace}}">
-                      <div class="icon-container" ng-style="{{category.style}}"><img class="icon" ng-src="images/sidebar-images/{{category.image}}" id="{{category.name | nospace}}"/></div> 
-                      <div class="category-main" id="{{category.name | nospace}}">{{category.name}}</div>
-                    </div>  
+                      <div class="icon-container" ng-style="{{category.style}}">
+                        <img class="icon" ng-src="images/sidebar-images/{{category.image}}"
+                             id="{{category.name | nospace}}"/>
+                      </div>
+                      <div class="category-main"
+                           id="{{category.name | nospace}}">{{category.name}}</div>
+                    </div>
                   </a>
                 </li>
               </ul>
           </div>
+=======
+>>>>>>> 0f6699c... Fixed html lint errors
         <div class="col-sm-3 col-md-2 sidebar">
           <div id="navSidebar">
             <ul class="nav nav-sidebar" isolateClick>
                 <div id="catTitle">
-                  <div class="icon-container-head"><img class="icon-header" ng-src="images/sidebar-images/menu.png"/></div> 
+<<<<<<< HEAD
+                  <div class="icon-container-head">
+                    <img class="icon-header" ng-src="images/sidebar-images/menu.png"/>
+                  </div>
+=======
+                  <div class="icon-container-head"><img class="icon-header" ng-src="images/sidebar-images/menu.png"></div>
+<<<<<<< HEAD
+                  <div class="category-main-header">
+<h4>Categories</h4>
+</div>
+=======
+>>>>>>> 0f6699c... Fixed html lint errors
                   <div class="category-main-header"><h4>Categories</h4></div>
+>>>>>>> add0f2f... Fixed html lint errors
                 </div>
                 <li ng-repeat="category in categoryKeys track by $index" class="">
                   <a href="#{{category.name | nospace}}" ng-click="categoryFilter($event)" id="{{category.name | nospace}}">
                     <div class="flex-container" id="{{category.name | nospace}}">
-                      <div class="icon-container" ng-style="{{category.style}}"><img class="icon" ng-src="images/sidebar-images/{{category.image}}" id="{{category.name | nospace}}"/></div> 
+<<<<<<< HEAD
+                      <div class="icon-container" ng-style="{{category.style}}">
+                        <img class="icon" ng-src="images/sidebar-images/{{category.image}}"
+                             id="{{category.name | nospace}}"/>
+                      </div>
+=======
+                      <div class="icon-container" ng-style="{{category.style}}"><img class="icon" ng-src="images/sidebar-images/{{category.image}}" id="{{category.name | nospace}}"></div>
```

Comment:
Line is longer than allowed. (187 > 129)

*LineLengthBear, severity NORMAL, section `default`.*